modules/dynamic-apiserver/pkg/generated/restapi/operations/backup/backup_list.go

```diff
@@ -31,12 +31,12 @@ func NewBackupList(ctx *middleware.Context, handler BackupListHandler) *BackupLi
 	return &BackupList{Context: ctx, Handler: handler}
 }
 
-/* BackupList swagger:route GET /backups/ backup backupList
+/*
```

Comment:
Why is it changed?

Reply:
Swagger version issues, now it's freezed in Makefile